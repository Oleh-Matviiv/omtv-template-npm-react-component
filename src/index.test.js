import React from 'react';
import renderer from 'react-test-renderer';
import Input from './';

describe('snapshots:', () => {
  it('renders correctly:', () => {
    const component = renderer.create(
      <Input />
    ).toJSON();
    expect(component).toMatchSnapshot();
  } );
});
