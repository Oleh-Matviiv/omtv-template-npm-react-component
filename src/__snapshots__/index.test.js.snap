// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots: renders correctly: 1`] = `
<input
  type="text"
/>
`;
